### DISCUSSION TITLE
Any chance we can get the src for the daisyui.com website?

### DISCUSSION THREAD

**Discussion by Crypt0Jesus** [saadeghi/daisyui#592] [Ideas / feature requests]
You have some wonderful examples on how to put together components.  I would really like to see how I can make some of the same, for example the stats tab.

**Comment by saadeghi** [comment]
daisyUI website source is here: https://github.com/saadeghi/daisyui/tree/master/src/docs  
You can also inspect the HTML code on your browser and see the class names or just copy them.

**Comment by Crypt0Jesus** [comment]
I'm not too bright.  I could have figured that out.  You sir rock.
